tensorbay/cli/utility.py

```diff
@@ -294,3 +294,16 @@ def is_accesskey(arg: str) -> bool:
 
     """
     return arg.startswith(("Accesskey-", "ACCESSKEY-")) and len(arg) == 42
+
+
+def shorten(origin: str) -> str:
+    """Reduce string's length to 7.
```

Comment:
This function does not change the original string, so the word `reduce` is not accurate.